### DISCUSSION TITLE
xUnit working example?

### DISCUSSION THREAD

**Discussion by pmcg86** [allure-framework/allure2#1588] [Questions & Support]
Hi,

As the title suggests, has anyone got a working example for Visual Studio and using xUnit?

The examples online are few and far between with very little insight of usage. I have downloaded the appropriate nuget packages and changed my Fact to AllureXunit etc but when I run dotnet test, the tests are not found. When i use the Test Runner the tests are ignored.

Also, when I attempt to run allure serve from the powershell terminal in VS it's saying JAVA_HOME is the incorrect directory (It's not, as I use allure with Jenkins and can also run allure commands from the command line outside of the project).

Any guidance would be great appreciated.

**Comment by pmcg86** [comment]
Surely someone must have a working example?

**Reply by sphinxy** [reply]
https://github.com/Tinkoff/Allure.XUnit/tree/master/src/Allure.XUnit.Examples this one works fine. 
Run tests directly from visual studio and it will create /bin/allure-results/*.json report.
